crates/cubecl-cuda/src/runtime.rs

```diff
@@ -230,6 +230,7 @@ impl Runtime for CudaRuntime {
 
         let mut sorted = strides.to_vec();
         sorted.sort();
+        sorted.reverse();
```

Comment:
Is it required?

Reply:
yes because the strides need to be smallest last not largest last. sort + reverse is the fastest way to get a vector sorted by largest first.